Permissions_UITest/Permissions_UITest/bin/Debug/Newtonsoft.Json.xml

```diff
@@ -2554,7 +2580,7 @@
             <summary>
             Gets or sets the maximum depth allowed when reading JSON. Reading past this depth will throw a <see cref="T:Newtonsoft.Json.JsonReaderException"/>.
             A null value means there is no maximum. 
-            The default value is <c>128</c>.
+            The default value is <c>64</c>.

```

Comment:
why so?

Reply:
It is bin/debug as you can see. Let's remove this stuff separately.